pylxd/tests/models/test_image.py

```diff
@@ -110,7 +111,8 @@ def test_create_with_metadata(self):
         """An image with metadata is created."""
         fingerprint = hashlib.sha256(b'').hexdigest()
         a_image = models.Image.create(
-            self.client, b'', metadata=b'', public=True, wait=True)
+            self.client, StringIO(u''), metadata=StringIO(u''),
```

Comment:
Instead of editing this test, can you please add another test that verifies the new functionality while allowing us to maintain a test to ensure no future regressions in the API handling?